pandas/core/arrays/boolean.py

```diff
@@ -656,6 +794,10 @@ def cmp_method(self, other):
         return set_function_name(cmp_method, name, cls)
 
     def _reduce(self, name, skipna=True, **kwargs):
+
+        if name in {"any", "all"}:
```

Comment:
we usually use lists for these checks

Reply:
In this file we actually use more `in {}` than `in []` (both are used), but since Tom and I wrote this file, that's probably not an argument ;) 
Happy to change it, purely performance wise the set is faster (but this is about nanoseconds of course ..)

Reply:
Heh, I'm probably to blame for the sets :) I like them more for membership tests, though it doesn't matter for small sets.